src/main/java/squeek/applecore/mixinplugin/CCLLegacyHelper.java

```diff
@@ -0,0 +1,71 @@
+package squeek.applecore.mixinplugin;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.tuple.Pair;
+
+import codechicken.core.launch.CodeChickenCorePlugin;
+
+public class CCLLegacyHelper {
+
+    public static boolean useLegacyMixin() {
+        return !useNewMixin();
+    }
+
+    @SuppressWarnings("deprecation")
+    public static boolean useNewMixin() {
+        String version;
+        try {
+            version = CodeChickenCorePlugin.version;
+        } catch (Exception e) {
+            return false;
+        }
+        return compareVersion("1.3.10", version);
+    }
+
+    /**
+     * @return {@code true} if {@code v2} is equal to or newer than {@code v1}.
+     */
+    private static boolean compareVersion(String v1, String v2) {
```

Comment:
Forge has a built in thing to compare versions `net.minecraftforge.fml.common.versioning.ComparableVersion`
any reason you don't use it other than not knowing it exists ?

Reply:
I was only aware of `ArtifactVersion` and thought it might be to early to load (although I never tested this). Will change.